src/app/shared/request/request.service.ts

```diff
@@ -150,29 +158,31 @@ export class RequestService {
       );
   }
 
-  post(endPoint: string = '', data, httpOptions?: RequestOptions, isLoginAPI?: boolean): Observable<any> {
-    if (!httpOptions) {
-      httpOptions = {};
+  post(params: POSTParams): Observable<any> {
```

Comment:
i like how this got cleaned up!

Reply:
Thank you @trtshen 👍. Need to do same refactoring for other functions like get in the future.